ephios/plugins/questionnaires/forms.py

```diff
@@ -3,11 +3,15 @@
 from django_select2.forms import Select2MultipleWidget
 
 from ephios.core.forms.events import BasePluginFormMixin
-from ephios.plugins.questionnaires.models import Question
+from ephios.plugins.questionnaires.models import Answer, Question
 
 
 class ChoiceForm(forms.Form):
-    name = forms.CharField(label=_("Choice name"))
+    name = forms.CharField(
+        label=_("Choice name"),
+        # pylint: disable=protected-access
+        max_length=Answer._meta.get_field("answer").max_length,
```

Comment:
I never thought about this before, but are the lengths considered the same for 2+ byte unicode chars in forms and the database? The database cares about bytes, but the for might care about actual character count, no?

Reply:
According to <https://docs.djangoproject.com/en/5.2/ref/models/fields/#django.db.models.CharField.max_length>, `max_length` on the model field also refers to the number of *characters*, so this should not be an issue